include/rijndael.h

```diff
@@ -18,9 +18,10 @@
 #ifndef RIJNDAEL_H
 #define RIJNDAEL_H
 
+typedef unsigned char   u8;
+typedef unsigned int    u32;
```

Comment:
I have no problem fixing this stuff up, I was thinking the same thing when I was exploring the code, but this is just me moving around what's already there, just to be clear. Fixes should come in a different commit.